Kitodo/src/test/java/org/kitodo/services/data/UserGroupServiceIT.java

```diff
@@ -218,4 +219,13 @@ public void shouldGetAuthorizationsAsString() throws Exception {
         List<String> expected = Arrays.asList("admin","manager","user");
         assertEquals("Permission strings doesn't match to given plain text!", expected, actual);
     }
+
+    @Test
+    public void shouldGetAuthorizationForAdmin() throws Exception {
+        List<UserGroupDTO> userGroupDTOS = userGroupService.convertJSONObjectsToDTOs(userGroupService.findByTitle("Admin", true), true);
+        assertEquals("Incorrect amount of found user groups", 1, userGroupDTOS.size());
+
+        AuthorizationDTO authorizationDTO = userGroupDTOS.get(0).getAuthorizations().get(0);
+        assertEquals("Incorrect authorization!","admin", authorizationDTO.getTitle());
```

Comment:
One white space between method parameters is needed.

Reply:
done